src/components/Card/Card.js

```diff
@@ -12,39 +13,55 @@ Card.propTypes = {
 }
 
 export default function Card({
+  id,
   text,
   title,
   image,
   address,
   openingHours,
   entry,
+  handleBookmark,
+  isFavorite,
 }) {
   const [isExpanded, setIsExpanded] = useState(false)
 
   return (
-    <WrapperCardList>
-      <img src={image} alt="" />
-      <h2>{title}</h2>
+    <>
```

Comment:
You do have the CardWrapper wrapping your whole card so no need for a fragment here. 
```suggestion

```